content/Accesibility in React: tools and practices.md

```diff
@@ -0,0 +1,152 @@
+
+- People who access with small screens cellphones
+
+- People using slow internet connection
+
+- And many more cases that we didn't know!
+
+You probably think after read this, that is impossible contemplate these kinds of disabilities for all the features that you implement every day, And that's true, is impossible create the perfect feature for everyone (in general, not only with accessibility subject), but is totally possible create a feature that the majority of the users can access to it, applying only a some practices and tools.
+
+## Introduce accessibility in your React application
+
+We have a lot of tools and standards out there, ready to be used and implemented in our project with only change little aspects in the way we develop every day. So in this section, there's an introduction of some of the tools and classics mistakes that React developers does that affect the accessibility of your application.
+
+### Fragments and the wrapper hell
+
+One of the classical mistakes that the react developers we commit usually is put a wrapper div in every component we created, for example:
+
+https://gist.github.com/nsantos16/8da556d82904423cab8e812a921d68fa
+
+This simple List component generates a div for each element, so when it rendering the `ListComponent` looks like this:
+
+https://gist.github.com/nsantos16/1c3159e118a2d8af754d2a57b5e732dd
+
+this makes the dom confusing for visual interpretations programs that are used by people with visual disabilities.
+
+A solution for this is using the API that React provides, more particularly, `</Fragment>`, let's see:
+
+https://gist.github.com/nsantos16/c097d8e791ad91ff45b0f835f5ab0219
+
+which rendering this:
+
+https://gist.github.com/nsantos16/eae10c9f74bcc8ccec3357165fa1b584
+
+### Put a title to every page
+
+Add a title to all the pages of your application allowing users to recognize what is the content of the page and if it is relevant for them. Overall, people with visual, auditory, severe mobility and short-term memory disabilities benefit from it.
+
+Furthermore, in React the use of **hooks** makes this an easy job:
+
+https://gist.github.com/nsantos16/d42e2c264971373834c3718be90995a4
+
+https://gist.github.com/nsantos16/02cee7facd1f13004c38c4def1deac46
+
+### Accessibility auditing with react-axe
+
+Integrate tools on the development process like linters, formatter codes, pre-commit hooks, and other things ensure code-quality and allow an agile implementation.
+
+[`react-axe`](<[https://github.com/dequelabs/react-axe](https://github.com/dequelabs/react-axe)>) is one of them, this linter/analyzer it will improve the process of include accessibility in your application in a fast way.
+
+This tool in his new version, apart of analyzing statically the code of your application, also _analyzes in run-time_, that means, if for example, the gradient of a screen is not sufficient for people with visual disabilities, the linter will throw a warning when you are running your application.
```

Comment:
```suggestion
This tool in its latest version, apart from statically analyzing the code of your application, also _analyzes in run-time_, that means that if, for example, the gradient of a screen is not sufficient for people with visual disabilities, the linter will throw a warning when you are running your application.
```

Reply:
`In its latest version, this tool analyzes your code, both statically and in run-time...`